docs/en/_sidebar.md

```diff
@@ -36,7 +35,7 @@
 
 - References
   - [Appfile](/en/developers/references/devex/appfile.md)
-  - [Capabilities](./developers/references/README.md)
+  - [Capabilities](/en/developers/references/README.md)
```

Comment:
This relative path is working and I have verified. @sunny0826 could you please double check it? Absolute path is not friendly for docs writer and to verify the link before deploying it to Github pages or running `docsify server`.

Reply:
Both ways of writing are valid